### ISSUE TITLE
Allow to set is_staff and is_superuser fields

### ISSUE THREAD

**Issue by jobec** [jobec/django-auth-adfs#16]
Right now it's not possible to properly set the `is_staff` and `is_superuser` fields on the user model of Django. Blocking access to the admin section of Django.

While there is a sort of workaround mentioned in #14 it should be handled in a better way.

**Comment by jobec** [comment]
First implementation in eee84872f85ef754a8abe8dac7b906298160c143

**Comment by jobec** [comment]
@backside180 Could you try the code from this commit and see if it works for you?
I've implemented a `BOOLEAN_CLAIM_MAPPING` setting ([see here](http://django-auth-adfs.readthedocs.io/en/latest/configuration.html#boolean-claim-mapping)).

**Comment by jobec** [comment]
Still missing an entry in the ADFS config guide in the docs.

**Comment by backside180** [comment]
@jobec This appears to be working fine for me.

Just to confirm my configuration to ensure I have implemented it as you expect after copying in the updated  backend.py and config.py files.

My claim rule is setting a value of "True" for an new field called "is_staff" based on the users membership of a given Admins AD group. 

In my Settings.py I removed the "is_staff" field mapping from the "CLAIM_MAPPING" dict and added it to "BOOLEAN_CLAIM_MAPPING" as follows 

> "BOOLEAN_CLAIM_MAPPING": {"is_staff": "is_staff"},

To test I removed my admin and minion test users from the django users list via the Admin area and after testing a login with both of these users I can confirm my "Admin" user has the "Staff Status"  and my "Minion" user does not. As expected.
